src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/details/details.html

```diff
@@ -57,13 +64,18 @@
 
     <div class="row">
       <div class="col-md-12">
-        <div class="detail-panel" id="hp-panel-detailed-info">
+        <div ng-if ="Utils.hasCrawlErrors()" class="detail-panel" id="hp-panel-detailed-info" >
+          <h4 class="record-title"
+            ng-bind-html="vm.record._extra_data.crawl_errors.file_name"
```

Comment:
shall we allow editing the ``file_name``?

Reply:
now it is allowed, if you check next line.

Reply:
I mean we should remove editing in this case, it doesn't make sense :)